### ISSUE TITLE
Provide user friendly command names in "Keyboard Shortcuts" view

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#1659]
### Describe the request

Convert all commands listed in the "**Keyboard Shortcuts**" view to the user friendly command name format.

🙂 The "**Keyboard Shortcuts**" view will be more readable and approachable to users.

### Describe the current behavior

Arduino IDE has a convenient GUI for discovering and customizing keyboard shortcuts. It consists of a table of "commands". The command names will be the user's only indication of what the keyboard shortcut does.

🙁 The names of many of the most important commands have a hyphenated form that is less approachable to the user.

#### To reproduce

1. Select **File > Advanced > Keyboard Shortcuts** from the Arduino IDE menus.
   A "**Keyboard Shortcuts**" view will open in the main panel.
1. In the "**Search keybindings**" field of the view, type `arduino`

### Arduino IDE version

3a70547

### Operating system

- Windows
- Linux

### Operating system version

- Windows 10
- Ubuntu 20.04

### Issue checklist

- [X] I searched for previous requests in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the feature was still missing when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My request contains all necessary details
